### DISCUSSION TITLE
[Microsoft Windows] Minimize to tray on start / start minimized

### DISCUSSION THREAD

**Discussion by patstha** [podman-desktop/podman-desktop#867] [Ideas]
On Microsoft Windows, it would be nice to have an option to minimize podman desktop on start so it doesn't open a new window every time I reboot the computer. 

![image](https://user-images.githubusercontent.com/56666486/202878522-dd4927e0-4df7-4459-a43b-b1c07964e68b.png)

Microsoft Teams calls it "open application in background"

![image](https://user-images.githubusercontent.com/56666486/202878621-07a422e7-81ae-4010-80e4-a9a498181638.png)

A user question and answer that explains this flow: https://techcommunity.microsoft.com/t5/microsoft-teams/allow-teams-application-start-minimized/m-p/1124951

**Comment by benoitf** [comment]
@patstha it looks a great improvement, moving as issue

**Comment by benoitf** [comment]
https://github.com/containers/podman-desktop/issues/868
